tox.ini

```diff
@@ -20,5 +35,5 @@ deps = sphinx==1.4.8
 whitelist_externals = make
 changedir = {toxinidir}/docs
 commands =
-    make clean
-    make html
+    rm -rf build
```

Comment:
The `make` commands work great locally, but they add an implicit dependency on `make` which doesn't fly on Travis: that's why Travis previously had the `sphinx-build` command instead of `make`.